images/ubuntu/scripts/build/waagent-deprovision-user.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+################################################################################
+##  File:  waagent-deprovision-user.sh
+##  Desc:  Runs user deprovision on azure instance
+################################################################################
+
```

Comment:
A good practice is to let the script fail if something doesn't work. Bash tends to continue executing even if some command fails.

```suggestion
# Set environment.
set -euxo pipefail
```